IntegrationTests/GoldMaster/BUILD

```diff
@@ -45,6 +46,298 @@ filegroup(
     "//visibility:public"
   ]
 )
+filegroup(
+  name = "FBSDKCoreKit_cxx_direct_hdrs",
+  srcs = glob(
+    [
+      "pod_support/Headers/Public/**/*"
+    ],
+    exclude_directories = 1
+  ),
+  visibility = [
+    "//visibility:public"
+  ]
+)
+filegroup(
+  name = "FBSDKCoreKit_cxx_public_hdrs",
+  srcs = [
+    ":Basics_public_hdrs",
+    ":Core_public_hdrs"
+  ],
+  visibility = [
+    "//visibility:public"
+  ]
+)
+filegroup(
+  name = "FBSDKCoreKit_cxx_hdrs",
+  srcs = glob(
+    [
+      "pod_support/Headers/Public/**/*"
+    ],
+    exclude_directories = 1
+  ),
+  visibility = [
+    "//visibility:public"
+  ]
+)
+filegroup(
+  name = "FBSDKCoreKit_cxx_union_hdrs",
+  srcs = [
+    "FBSDKCoreKit_cxx_hdrs",
+    "FBSDKCoreKit_hdrs",
+    ":Basics_hdrs",
+    ":Core_hdrs"
+  ],
+  visibility = [
+    "//visibility:public"
+  ]
+)
+headermap(
+  name = "FBSDKCoreKit_cxx_hmap",
+  namespace = "FBSDKCoreKit",
+  hdrs = [
+    "FBSDKCoreKit_package_hdrs",
+    ":FBSDKCoreKit_cxx_union_hdrs"
+  ],
+  deps = [
+    ":Basics_hmap",
+    ":Core_hmap"
+  ],
+  visibility = [
+    "//visibility:public"
+  ]
+)
+gen_includes(
+  name = "FBSDKCoreKit_cxx_includes",
+  include = [
+    "Vendor/FBSDKCoreKit/pod_support/Headers/Public/"
+  ]
+)
+objc_library(
```

Comment:
Could you explain why the names in `srcs` and then immediately `exclude`'ed?

Reply:
It uses `glob` as a [nandgate style operator to do addition/subtraction of the sources](https://github.com/pinterest/PodToBUILD/blob/master/Sources/PodToBUILD/ObjcLibrary.swift#L541). Currently, there's a bug in the `GlobNode.toSkylark` code that isn't nooping when `include != exclude`

Reply:
Is that a bug that still needs to be fixed, or is that what you're fixing here?

Reply:
Fixed here: https://github.com/pinterest/PodToBUILD/pull/153